lib/render/CMakeLists.txt

```diff
@@ -3,12 +3,12 @@ add_library(AirshipRenderer)
 target_include_directories(AirshipRenderer
     PUBLIC
         $<INSTALL_INTERFACE:include>
-        $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/lib/include>
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
 )
-target_sources(AirshipRenderer PUBLIC color.cpp)
+target_sources(AirshipRenderer PUBLIC src/render/color.cpp)
```

Comment:
For futureproofing, this should probably be done in render, with `add_subdirectory(src/render)` here.

Reply:
The pattern I want to go with here is that we have CMakeLists for each library, and to minimize the number of CMakeLists files beyond that point. So basically each library has the following layout:

include/
src/
CMakeLists.txt

And everything needed to build the library exists inside of that one CMakeLists file. If things get too complicated for a single library, it might be a good idea to split that library into smaller pieces. My main goal with this is to limit the number of places we have to track down where files are included in the builds so that if you add a new file there is no confusion which CMakeLists file it needs to be added to. The answer is always "the one at the same level as the src/ and include/ directories for said library.